too-many-hooks/src/useTally.ts

```diff
@@ -68,34 +68,34 @@ export type UseCountReturn = [
 ]
 
 /**
- * `useCount` hook type
+ * `useTally` hook type
  *
  * @export
- * @typedef {UseCount}
- * @param {UseCountArgs}
- * @returns {UseCountReturn}
+ * @typedef {UseTally}
+ * @param {UseTallyArgs}
+ * @returns {UseTallyReturn}
  */
-export type UseCount = (args: UseCountArgs) => UseCountReturn
+export type UseTally = (args: UseTallyArgs) => UseTallyReturn
 
 /**
- * Returns a numeric `count` and functions to set, increment, decrement, or reset it
+ * Returns a numeric `tally` and functions to set, increment, decrement, or reset it
  *
  * @example
- * To count up by 2s
+ * To increment by 2s
  * ```ts
- * const [count, { increment }] = useCount({step: 2})
+ * const [tally, { increment }] = useTally({step: 2})
  * ```
  * @example
- * To count down by 1s
+ * To decrement by 1s
  * ```ts
- * const [count, { decrement }] = useCount()
+ * const [tally, { decrement }] = useTally()
  * ```
- * @implements {UseCount}
- * @param {UseCountArgs} { initial = 0, step = 1 }
- * @returns {UseCountReturn}
+ * @implements {UseTally}
+ * @param {UseTallyArgs} { initial = 0, step = 1 }
+ * @returns {UseTallyReturn}
  */
-export const useCount: UseCount = ({ initial = 0, step = 1 }: UseCountArgs): UseCountReturn => {
-  const [count, set] = useState<number>(initial)
+export const useTally: UseTally = ({ initial = 0, step = 1 }: UseTallyArgs): UseTallyReturn => {
+  const [tally, set] = useState<number>(initial)
 
   const increment = useCallback(
     (stepOverride?: number) => set(c => c + (stepOverride ?? step)),
```

Comment:
```suggestion
    (stepOverride?: number) => set(t => t + (stepOverride ?? step)),
```
We should also update these since `c` was short for `count`. Same goes for the `decrement`.